packages/@glimmer/compiler/test/compiler-test.ts

```diff
@@ -50,8 +51,8 @@ function test(desc: string, template: string, ...expectedStatements: BuilderStat
   });
 }
 
-const Append = Builder.Append;
-const Concat = Builder.Concat;
+const Append = BUILDER_APPEND;
```

Comment:
Why alias these?

It's how it was, and unaliasing them should probably be a separate PR

Reply:
These are entirely internal aliases in this test file. It makes sense to unalias them, absolutely.